common/components/utils/NavigationLinks.js

```diff
@@ -27,8 +21,7 @@ const siteNavLinks =
 class NavigationLinks {
   static list(): $ReadOnlyArray<NavigationLink> {
     try {
-      const envLinks: $ReadOnlyArray<DLAB_NAV_LINK> = siteNavLinks;
-      return envLinks.map( (link:DLAB_FOOTER_LINK) => ({url:link.u, name:link.n, isButton:link.isButton}));
+      return siteNavLinks.map( (link:NavigationLink) => ({url:link.u, name:link.n, isButton:link.isButton}));
```

Comment:
Just format siteNavLinks the way it's going to be used, and we don't need to map.